openjdk8/pom.xml

```diff
@@ -81,7 +81,7 @@
 	    <configuration>
 	      <imageName>openjdk8</imageName>
```

Comment:
OK - but can we bump that rename to a different issue/PR as it probably comes into the version/buildID considerations.  

We may even need multiple names/tags for the one image as many users will want to get the latest java8 with `openjdk8:latest`

So for now can we strip out the unneeded async- prefix?